src/main/java/com/labsynch/labseer/service/StandardizationServiceImpl.java

```diff
@@ -767,5 +736,52 @@ public StandardizationHistory getMostRecentStandardizationHistory() {
 			return null;
 		}
 	}
+
+	public void finalizeDryRun(String status) throws StandardizerException {
+		//Check for any parent ids not in dry run table - This varifies that population completed.
+		//Check if all dry run compounds have...
+		//  -- getRegistrationStatus is ERROR
+		//  -- or dryRunCompound changedStructure is filled in
+		StandardizationHistory stndznHistory = getMostRecentStandardizationHistory();
+		logger.info("Most recent standardization history id: " + stndznHistory.getId() + " status: " + stndznHistory.getDryRunStatus());
+		//Refresh the stndznHistory to get the latest status
+        if (!stndznHistory.getDryRunStatus().equals("running")) {
+            return;
+        }
+		stndznHistory.setDryRunComplete(new Date());
+		stndznHistory.setDryRunStatus(status);
+		logger.info("Setting standardization history id " + stndznHistory.getId() + " to " + status);
+		stndznHistory
+				.setDryRunStandardizationChangesCount(StandardizationDryRunCompound.getReadyStandardizationChangesCount());
+		stndznHistory.merge();
+		logger.info("Set dry run to " + status);
+	};
+		
+
+
+	@Scheduled(fixedDelayString = "${client.cmpdreg.serverSettings.checkForDryRunStandardizationDelay:60000}")
+    public void checkForDryRunStandardization() throws CmpdRegMolFormatException, IOException, StandardizerException {
```

Comment:
This is the actual scheduled task which defaults to 60 seconds. 
It checks to see if standardization is already running on this worker and exits if it is or sets it running if it is not.
 It checks the most recent standardization history to see if it's running, and if it is, it kicks off `runDryRun` which helps populate the the table, run deduplication and finalizes the run.